website/blog/2022-10-18-how-to-design-metrics.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "How to design metrics: deciding what to model in the modeling layer vs the semantic layer"
+description: "placeholder"
+slug: how-to-design-metrics
+
+authors: [callum_mccann]
+
+tags: [dbt product updates]
+hide_table_of_contents: false
+
+date: 2022-10-18
+is_featured: true
+---
+
+---
+
+**IMPORTANT:** This document serves as the temporary location for information on how to design and structure your metrics. It is our intention to take this content and turn it into a Guide, similar to [How we structure our dbt projects](https://docs.getdbt.com/guides/best-practices/how-we-structure/1-guide-overview) , but we feel that codifying information in a Guide first requires that Metrics be rigorously tested by the community so that best practices can arise. This document contains our early attempts to create best practices. In other words, read these as suggestions for a new paradigm and share in the community where they do (or don’t) match your experiences! More information on where to do this can be found at the end.
+
+---
+
+## The power of a semantic layer on top of a mature data modeling framework
+
+As a longtime dbt Community member, I knew I had to get involved when I first saw the dbt Semantic Layer. It gave me a vision of a world where metrics and business logic were unified across an entire organization; a world where the data team was no longer bound to a single consuming experience and could enable their stakeholders in dozens of different ways. To me, it felt like the opportunity to contribute to the next step of what dbt could become.
+
+In past roles, I’ve been referred to as the `dbt zealot` and I’ll own that title! But it’s not a surprise - dbt was built to serve analytics engineers exactly like me. It gave us flexibility and power to serve our organizations. But it always felt like dbt best served **us.**
```

Comment:
```SUGGESTION
In past roles, I’ve been referred to as the `dbt zealot` and I’ll gladly own that title as dbt was built to allow data practitioners like me to greatly expand the power of our work by writing code based off of software engineering principles. It gave us flexibility and power to serve our organizations. But I always wondered if there were more folks who could directly benefit from interacting with dbt.

Reply:
made some changes but kept the first part because I like the flow of a hard cut after "I'll own that title!"